druid/ingestion-spec/rollups/ml_project_index_rollup.json

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
Add all these ingestion specs under this template directory - https://github.com/project-sunbird/sunbird-data-pipeline/tree/release-4.9.0/ansible/roles/druid-ingestion/templates